dist/redhat/scylla-jmx.spec.mustache

```
@@ -41,6 +41,7 @@ fi
 
 %post
 %systemd_post scylla-jmx.service
+/usr/bin/systemctl daemon-reload
```

Will this fail in docker where systemd is not installed?

---

Should have `|| :` at the end to suppress errors. https://docs.pagure.org/packaging-guidelines/Packaging%3AScriptlets.html

---

Updated the code: https://github.com/scylladb/scylla-jmx/pull/92/files#diff-05421edf403fa53414100044a254258eR44